plugins/module_utils/api.py

```diff
@@ -189,3 +192,23 @@ def _format_user_agent(collection_version, python_version=None, ansible_version=
         py_version=python_version or "unknown",
         ansible=ansible_version or "unknown"
     )
+
+
+# Client UUIDs must be exactly 26 characters.
+CLIENT_UUID_LENGTH = 26
+
+
+def valid_client_uuid(uuid):
+    if len(uuid) > CLIENT_UUID_LENGTH:
```

Comment:
Could you please explain why do we allow uuid to be shorter, and we only check for whether it is longer?
Does the below regex match take care of that?

Reply:
Yes, the regex below will ensure the string is exactly 26 characters long. I see your point, though, so I'll get rid of the check here and just rely on the regex.

Confirmation: https://regex101.com/r/NrLJT6/1